src/StripeTests/StripeTests.csproj

```diff
@@ -13,6 +13,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
```

Comment:
I assume that you didn't add this on purpose (it's not necessary since we already import a more recent version of the same package just two lines below). Can you remove the line and run `dotnet restore` from the root of the repo to ensure that the new version is installed?

Reply:
Weird, I likely messed up my rebase. Thanks for catching, will fix.

Reply:
fixed